langserver/config.go

```diff
@@ -29,27 +30,16 @@ import (
 // Config contains the configuration for a server.
 type Config struct {
 	RPCTrace      string `yaml:"rpc_trace"`
+	LogFormat     string `yaml:"log_format"`
 	PrometheusURL string `yaml:"prometheus_url"`
 	RESTAPIPort   uint64 `yaml:"rest_api_port"`
 }
 
-func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
This method was here to override the yaml unmarshalling. By removing it, you don't validate the config when you are passing the config through a file

Reply:
Thanks for the explanation.

I've added a comment so it doesn't accidentally removed again.